Admin/Admin.php

```diff
@@ -416,6 +416,13 @@
     protected $breadcrumbs = array();
 
     /**
+     * The breadcrumbsBuilder component.
+     *
+     * @var BreadcrumbsBuilderInterface
+     */
+    private $breadcrumbsBuilder;
```

Comment:
According to Symfony Standards, private properties should be placed after protected ones.